.circleci/config.yml

```diff
@@ -310,14 +310,9 @@ defaults:
         CXX: clang++
         MAKEFLAGS: -j 3
 
-  - base_ubuntu2004_clang_small: &base_ubuntu2004_clang_small
+  - base_ubuntu2004_clang_medium: &base_ubuntu2004_clang_medium
```

Comment:
I just realized that we already have a definition for `medium`. It's `base_ubuntu2004_clang` above (`medium` is the default). So you actually don't have to add a new definition. Just use that one.

Reply:
That sounds better. Will remove the definition entirely in the latest push.